starport/services/chain/serve.go

```diff
@@ -504,10 +504,6 @@ func (c *Chain) watchAppFrontend(ctx context.Context) error {
 				step.Env(
 					fmt.Sprintf("HOST=%s", host),
 					fmt.Sprintf("PORT=%s", port),
-					fmt.Sprintf("VUE_APP_API_COSMOS=%s", xurl.HTTP(conf.Host.API)),
```

Comment:
Why we should rm these? Removing them will make the frontend get broken on custom host configuration the config.yml and force users to enter env vars manually.

Reply:
see discussion above and here: https://github.com/tendermint/starport/issues/1006

Reply:
the idea is that custom configuration merits explicit initialization of the vuex/env module with the right URLs and we only cater to defaults